manager/src/main/java/org/apache/hertzbeat/manager/component/alerter/DispatcherAlarm.java

```diff
@@ -129,6 +130,9 @@ public void run() {
                     if (alert != null) {
                         // Determining alarm type storage
                         alertStoreHandler.store(alert);
+                        if (alert.getTags() != null && alert.getTags().containsKey(CommonConstants.TAG_COLLECTOR_NAME)){
+                            continue;
+                        }
```

Comment:
hi, here has a question that why ignore to send notify the collector alarm message

Reply:
`AlertNotifyHandler` and`Monitor` are heavily coupled and the changes are large. Therefore, `send notify` is skipped for now

Reply:
Got it👍 It is recommended that we can merge after version 1.6.1 and then see how to solve the coupling problem between monitor and alarm.